### DISCUSSION TITLE
Sync error(WebDAV)

### DISCUSSION THREAD

**Discussion by feroooooo** [remotely-save/remotely-save#161] [Q&A] [answered]
When I use Wi-Fi,it works normally.But when I use 4G cellular data, it doesn't work. My English is not well, sorry for my weird expressions.
![image](https://user-images.githubusercontent.com/109299163/181571339-67b54f47-a607-429c-99f8-ee5fdff142b7.jpeg)

**Comment by fyears** [comment, marked as answer]
Sorry but would you like to check whether your webdav server has something like api restrictions?
